src/Tribe/Views/V2/Customizer.php

```diff
@@ -32,6 +34,128 @@ public function filter_sections( array $sections, $customizer ) {
 		return $sections;
 	}
 
+	/**
+	 * Adds new settings/controls via the hook in common.
+	 *
+	 * @since TBD
+	 *
+	 * @param \Tribe__Customizer__Section $section    The Global Elements Customizer section.
+	 * @param WP_Customize_Manager        $manager    The settings manager.
+	 * @param \Tribe__Customizer          $customizer The Customizer object.
+	 */
+	public function include_global_elements_settings( $section, $manager, $customizer ) {
+		// Event Title.
+		$manager->add_setting(
+			$customizer->get_setting_name( 'event_title_color', $section ),
+			[
+				'default'              => '#141827',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_title_color', $section ),
+				[
+					'label'    => esc_html__( 'Event Title', 'the-events-calendar' ),
+					'section'  => $section->id,
+					'priority' => 8,
+				]
+			)
+		);
+
+		// Event Date & Time.
+		$manager->add_setting(
+			$customizer->get_setting_name( 'event_date_time_color', $section ),
+			[
+				'default'              => '#141827',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_date_time_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Date and Time', 'the-events-calendar' ),
+					'description' => esc_html__( 'Main date and time display on views and single event pages.', 'the-events-calendar' ),
+					'section'     => $section->id,
+					'priority'    => 8,
+				]
+			)
+		);
+
+		// Background Color.
+		$manager->add_setting(
+			$customizer->get_setting_name( 'background_color_choice', $section ),
+			[
+				'default'              => 'transparent',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_key',
+				'sanitize_js_callback' => 'sanitize_key',
+			]
+		);
+
+		$manager->add_control(
+			new Control(
+				$manager,
+				$customizer->get_setting_name( 'background_color_choice', $section ),
+				[
+					'label'       => 'Background Color',
+					'section'     => $section->id,
+					'description' => esc_html__( 'All calendar and event pages.', 'the-events-calendar' ),
+					'type'        => 'radio',
+					'priority'    => 12,
+					'choices'     => [
+						'transparent' => esc_html__( 'Transparent', 'the-events-calendar' ),
+						'custom'      => esc_html__( 'Select Color', 'the-events-calendar' ),
+					],
+				]
+			)
+		);
+
+		$manager->add_setting(
+			$customizer->get_setting_name( 'background_color', $section ),
+			[
+				'default' => '#fff',
+				'type'    => 'option',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 17 space(s) between "'type'" and double arrow, but found 4.

<sub>hash: b0574a29c4eac6afaef1502dad032f33</sub>